packages/space-opera/src/components/utils/render_model_viewer.ts

```diff
@@ -95,7 +95,7 @@ export function renderModelViewer(
     childElements?: Array<TemplateResult|HTMLElement>) {
   const skyboxImage =
       config.useEnvAsSkybox ? config.environmentImage : undefined;
-  return html`<model-viewer ...=${spread(extraAttributes)}
+  return html`<model-viewer ${spread(extraAttributes)}
```

Comment:
Thanks! This was the part I was struggling with. I've done some testing locally and it all seems to work great, but I noticed one odd thing that I can't seem to find the source of: 
<img width="355" alt="image" src="https://user-images.githubusercontent.com/1649964/166338230-4aacf27b-6ba5-42fe-8900-8af243a6f446.png">
Specifically this: `<!--?lit$515089429$-->`
Not that it's really hurting anything, but it's ugly and slightly ominous...

Reply:
Ah, that's new in Lit 2, used for keeping track of Lit template binding locations. I can see if I can scratch something up to try and strip them from that textarea if you'd like?

Reply:
Yes, that'd be very helpful. This is user-facing, so I'd prefer not to confuse them with internal details.

Reply:
done! see:

[`0de10eb` (#3405)](https://github.com/google/model-viewer/pull/3405/commits/0de10eb0a3fb7f93999beb8f7a5cab824b7b6897)